src/content/docs/en/guides/astro-db.mdx

```diff
@@ -246,6 +426,33 @@ export const DELETE: APIRoute = async (ctx) => {
 }
 ```
 
+The Actions feature provides a more straightforward approach for inserting data into Astro DB. Here's an example of how to perform a database insert using Actions:
+
+```ts
+// src/actions/index.ts
+import { db, Comment } from 'astro:db';
+import { defineAction } from 'astro:actions';
+import { z } from 'astro:schema';
+
+export const server = {
+  addComment: defineAction({
+    // Actions include type safety with Zod, removing the need
+    // to check if typeof {value} === 'string' in your pages
+    input: z.object({
+      author: z.string(),
+      content: z.string(),
+    }),
+    handler: async (input) => {
+      const updatedComments = await db
+        .insert(Comment)
+        .values(input)
+        .returning(); // Return the updated comments
+      return updatedComments;
+    },
+  }),
+};
+```
```

Comment:
Did you get a chance to confirm this example works? @JusticeMatthew mentioned it was probably worth double checking since he hasn't been using Actions for very long.

Reply:
Honestly, no, but I can check right now. I intended to write a brief message about what has changed and what needs to be looked at, but I got sidetracked with other tasks.

Reply:
it works!
https://stackblitz.com/edit/github-qteuef (everything else apart from the action is done by bold.new. its pretty good!)